Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-interfaces.controller.ts

```diff
@@ -1,142 +1,45 @@
-                    dataRow.dataType = _.find(dataTypes, { Id: dataRow.dataTypeId });
-                });
+    app.controller("systemUsage.ExposedInterfaces", ["$scope", "itSystemUsage", "exhibits", "user", "lazyLoadingService",
+        ($scope, itSystemUsage, exhibits, user, lazyLoadingService) => {
 
-                theInterface.numRows = theInterface.dataRows.length;
-            }
+            $scope.systemUsage = itSystemUsage;
 
-            // interface exposures
-            _.each(exhibits, function (interfaceExposure: { updateUrl; urlParams; id; }) {
+            // Enables marking "wished" 
+            _.each(exhibits, (interfaceExposure: { updateUrl; urlParams; id; }) => {
                 interfaceExposure.updateUrl = "api/itInterfaceExhibitUsage/";
-                interfaceExposure.urlParams = "&usageId=" + itSystemUsage.id + "&exhibitId=" + interfaceExposure.id;
-                resolveTypes(interfaceExposure);
+                interfaceExposure.urlParams = `&usageId=${itSystemUsage.id}&exhibitId=${interfaceExposure.id}`;
             });
-            $scope.interfaceExposures = exhibits;
-
-            $scope.itSystemUsageSelectOptions = selectLazyLoading("api/itSystemUsage", ["organizationId=" + itSystemUsage.organizationId]);
-            function selectLazyLoading(url, paramAry) {
-                return {
-                    allowClear: true,
-                    minimumInputLength: 1,
-                    initSelection: function (elem, callback) {
-                    },
-                    ajax: {
-                        data: function (term, page) {
-                            return { query: term };
-                        },
-                        quietMillis: 500,
-                        transport: function (queryParams) {
-                            var extraParams = paramAry ? "&" + paramAry.join("&") : "";
-                            var res = $http.get(url + "?q=" + queryParams.data.query + extraParams).then(queryParams.success);
-                            res.abort = function () {
-                                return null;
-                            };
-
-                            return res;
-                        },
-
-                        results: function (data, page) {
-                            var results = [];
 
-                            _.each(data.data.response, function (obj: { id; itSystem; }) {
-                                results.push({
-                                    id: obj.id,
-                                    text: obj.itSystem.name
-                                });
-                            });
+            $scope.interfaceExposures = exhibits;
 
-                            return { results: results };
-                        }
-                    }
-                };
-            }
+            $scope.itSystemUsageSelectOptions = lazyLoadingService.selectLazyLoading("api/itSystemUsage", [`organizationId=${user.organizationId}`]);
```

Comment:
Væk med den både her og IT-system,,, + servicen. Det lader ikke til at være anvendt